corehq/apps/users/migrations/0031_index_userhistory_details.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.24 on 2021-07-17 00:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0030_userhistory_user_upload_record'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='userhistory',
+            index=models.Index(fields=['details'], name='users_userh_details_2679d6_idx'),
```

Comment:
are we still keeping the index? I thought this was not getting used on searching.

Reply:
oops, forgot to push changes, thanks